joomla/joomla-cms
# User notes search tools ordering & bug corrections

#### Description

This PR adds, for consistency, as it exists in so many other views in the backoffice, the order select box of the search tools to com_users notes.

Also corrects the bug that made search tools has always selected (open), even if no filter has selected and the pre-selected ordering has different in the xml and in the php.

And other small code changes.
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/12931492/1a3111bc-cf76-11e5-9ac4-5c8df936a6e0.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/12931452/dab71342-cf75-11e5-8eda-9bf903e55d4a.png)
#### How to test
1. Install latest staging and apply this patch
2. Go to Users -> User Notes
3. Check the ordering select box and the filter are working fine.
4. Check in the modal view too (create some notes for a user and then go to user list and there is a button to view user notes).
